prime-router/docs/universal-pipeline/route.md

```diff
@@ -255,15 +268,15 @@ The processing mode of the data indicates the sender’s intended context for th
  *  Must have a processing mode id of 'P'
  */
 val processingModeFilterDefault: ReportStreamFilter = listOf(
-    "%processingId = 'P'"
-)
+        "%processingId = 'P'"
+    )
 ```
 
 ### Condition Filter
 
-Filter data based on the test identifiers. A receiver expecting flu results should only accept tests for flu. If the message contains multiple observations, some that pass the condition filter and others that do not, the condition filter will be used to identify the desired observations. Identifiers for the needed observations are added to the Endpoint which is then added to the Provenance resource.
-
-*The Translate step will review the Endpoints and remove any observations that are not identified.
+Filter data based on the test identifiers. A receiver expecting flu results should only accept tests for flu. If the
+message contains multiple observations, some that pass the condition filter and others that do not, the condition filter
+will pass if any the observations are of interest to the receiver.
```

Comment:
Should we add a sentence to mention that observations not matching will be removed here? Before we mentioned that it was part of the translate step, so even though it's in routing now, we might still want to mention it.

Reply:
Yes, I'll update this